backend/composer/models.py

```diff
@@ -269,6 +325,18 @@ def name(self):
             return f"{region_name},{layer_name}"
         else:
             return "Unnamed Entity"
+        
+    @property
+    def ontology_uri(self):
+        if self.simple_entity:
+            return self.simple_entity.ontology_uri
```

Comment:
suggestion: @D-GopalKrishna why not do something Pythonic like this: `return simple if simple else region layer`

Reply:
Yes indeed, already did - https://github.com/MetaCell/sckan-composer/pull/252#issuecomment-2031215692. And merged to this branch as well.